daemon/daemon_main.go

```diff
@@ -670,6 +670,9 @@ func init() {
 	flags.Int(option.ToFQDNsMaxIPsPerHost, defaults.ToFQDNsMaxIPsPerHost, "Maximum number of IPs to maintain per FQDN name for each endpoint")
 	option.BindEnv(option.ToFQDNsMaxIPsPerHost)
 
+	flags.StringVar(&option.Config.ToFQDNsPreCache, option.ToFQDNsPreCache, defaults.ToFQDNsPreCache, "DNS cache data at this path is preloaded on agent startup")
```

Comment:
ah, so you are the one that keeps adding the StringVar thingy. You can use the `flags.String(` which doesn't need you to pass `option.Config.ToFQDNsPreCache`. Jut make sure this is assigned like all flags under `pkg/config/config.go` on lines 990s

Reply:
oh, I just copy-pasted another thing... I'll switch it :):